index.ts

```diff
@@ -286,28 +286,47 @@ TEST: addTests('isQuickPR', [
 	'https://github.com/sindresorhus/refined-github/compare/test-branch?quick_pull=1',
 ]);
 
-const prStateSelector = [
+const stateSelector = [
 	'.State',
 	'[class^="StateLabel"]',
 ].join(',');
 
-export const isDraftPR = (): boolean => $(prStateSelector)!.textContent!.trim() === 'Draft';
+export const isDraftPR = (): boolean => isPR() && $(stateSelector)!.textContent!.trim() === 'Draft';
 export const isOpenPR = (): boolean => {
-	const status = $(prStateSelector)!.textContent!.trim();
+	if (isPR()) {
+		return false;
+	}
+
+	const status = $(stateSelector)!.textContent!.trim();
 	return status === 'Open' || status === 'Draft';
 };
 
 export const isMergedPR = (): boolean => {
-	const status = $(prStateSelector)!.textContent!.trim();
+	if (isPR()) {
+		return false;
+	}
+
+	const status = $(stateSelector)!.textContent!.trim();
 	return status === 'Merged';
 };
 
 export const isClosedPR = (): boolean => {
-	const status = $(prStateSelector)!.textContent!.trim();
+	if (isPR()) {
+		return false;
+	}
+
+	const status = $(stateSelector)!.textContent!.trim();
 	return status === 'Closed' || status === 'Merged';
 };
 
-export const isClosedIssue = (): boolean => exists('#partial-discussion-header :is(.octicon-issue-closed, .octicon-skip)');
+export const isClosedIssue = (): boolean => {
+	if (isIssue()) {
+		return false;
+	}
+
+	const status = $(stateSelector)!.textContent!.trim();
+	return status === 'Closed';
```

Comment:
It needs to account for the other "Closed as unplanned" state

Reply:
Hhhh, my bad. 😬